### PR TITLE
changed ssh_private_key to ssh-private-key to match greymatter init sync

### PR DESCRIPTION
**Pull request by alecholmez**
Signed-off-by: Alec Holmes <alecholmez@me.com>

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1,4 +1,4 @@
 [submodule "pkg/cuemodule/core"]
 	path = pkg/cuemodule/core
-	branch = main
+	branch = private-key
```
Comment (danielpcox) [inline .gitmodules:3]: Reminder to switch this back

**Comment by danielpcox** [comment]
Probably want to bump the submodule in this branch now that you've merged the companion PR in gitops-core

File: .gitmodules (hunk 1 above)
Comment (alecholmez) [reply .gitmodules:3]: Ah yes thank you!

**Review by danielpcox — APPROVED** [review]
Lookin' good, if the integration tests pass MERGE MERGE MERGE!!

**Pull request merged**